src/components/element-editor/validation/validation.tsx

```diff
@@ -55,10 +55,9 @@ export const Validation = () => {
         if (childEntryValues && childEntryValues.length > 0) {
             try {
                 const validationSchema = schema.pick(["validation"]);
-                const validationValues = getValues("validation");
 
                 validationSchema.validateSync({
-                    validation: validationValues,
+                    validation: childEntryValues,
                     abortEarly: false,
                 });
                 return false;
```

Comment:
let's try to use the values from RHF directly, store the values in a local variable and then use it through this component

`childEntryValues`'s role now is only to trigger a re-render, when a re-render happens, the local variable in this component will be refreshed with the latest values

Reply:
this is not done yet?

Reply:
updated in the latest changes